examples/stateful_example.yaml

```
@@ -0,0 +1,115 @@
+  # Session window aggregation stream
+  - input:
+      type: kafka
+      brokers: ["localhost:9092"]
+      topics: ["user-events"]
+      consumer_group: "session-aggregator"
+    pipeline:
+      thread_num: 4
+      processors:
+        - type: sql
+          query: |
+            SELECT 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)